weblab/templates/entities/entity_runexperiments.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load entities %}
+
+{% block title %}Experiment: run specific combinations - {% endblock title %}
+
+{% block body_id %}runexperiment{% endblock %}
+
+{% block content %}
+  <h1 id="entityname">Run experiments using <span>{{ entity.name }}</span></h1>
+    <!-- SK TO DO: make these work -->
+     <p>
+         <input class="overrideCheckBox" type="checkbox" value="false"/>
+          Force overwrite of existing experiments
+     </p>
+  <div id="runexperiment">
+
+
+    <!-- SK TO DO: make these work -->
+     <button id="checkallbutton">check all</button>
+    <button id="checklatestbutton">check latest</button>
+    <button id="uncheckallbutton">uncheck all</button>
+   <p>
+        <form action="" method="post">
+      {% csrf_token %}
+       <button id="savebutton">run experiments</button>
```

Comment:
We might want to make this a normal Django form submission rather than an AJAX callback; not sure? Keeping closer to what WL1 did might be easier! Up to you :)

Reply:
I was modelling this on what I found many other views did. I haven't come across a button that does something more than redirect that does not use this form; but I may have missed one :-)

Reply:
Indeed, I think this is the typical form structure for an AJAX submission. Probably like the `saveNewVersion` function in editMetadata.js; I dare say there are others. I was just wondering if it wanted to be more like `EntityNewVersionView` with a standard HTML form - but that would require more changes than matching the WL1 approach, so probably not worth doing.

Reply:
Let me stick with whats working for me at present and then if/when we get round to tackling Michael's many issues we can add some clean up.